.env.sample

```diff
@@ -2,14 +2,23 @@
 NEXT_PUBLIC_ENVIRONMENT=dev
 
 # IPFS
-NEXT_PUBLIC_PINATA_JWT=###
-NEXT_PUBLIC_IPFS_READ_GATEWAY=###
+## get at https://app.pinata.cloud/developers/api-keys
+## get at https://app.pinata.cloud/gateway
+## get at https://app.pinata.cloud/onboarding
+
+NEXT_PUBLIC_PINATA_JWT=### 
+NEXT_PUBLIC_IPFS_READ_GATEWAY=### 
 NEXT_PUBLIC_IPFS_WRITE_GATEWAY=###
 
 # PROVIDER
+
+## get at https://www.infura.io/
 INFURA_ID=###
+## get at https://www.alchemy.com/
 ALCHEMY_ID=###
+## get at https://cloud.walletconnect.com/
```

Comment:
i *think* this is a walletconnect project id?  but it could be something else?

Reply:
im getting a 
```
TypeError: Only absolute URLs are supported
    at getNodeRequestOptions (webpack-internal:///(rsc)/./node_modules/node-fetch/lib/index.mjs:1196:15)
    at eval (webpack-internal:///(rsc)/./node_modules/node-fetch/lib/index.mjs:1292:25)
    at new Promise (<anonymous>)
    at fetch (webpack-internal:///(rsc)/./node_modules/node-fetch/lib/index.mjs:1289:12)
    at fetch (webpack-internal:///(rsc)/./node_modules/cross-fetch/dist/node-ponyfill.js:10:22)
    at eval (webpack-internal:///(rsc)/./node_modules/graphql-request/build/esm/index.js:123:22)
    at makeRequest (webpack-internal:///(rsc)/./node_modules/graphql-request/build/esm/index.js:267:28)
    at GraphQLClient.request (webpack-internal:///(rsc)/./node_modules/graphql-request/build/esm/index.js:177:16)
    at request (webpack-internal:///(rsc)/./node_modules/graphql-request/build/esm/index.js:314:19)
    at getPools (webpack-internal:///(rsc)/./src/app/page.tsx:48:95)
    at Home (webpack-internal:///(rsc)/./src/app/page.tsx:74:33)
    at /Users/owocki/Documents/SeaGrants/node_modules/next/dist/compiled/next-server/app-page.runtime.dev.js:38:9075
    at /Users/owocki/Documents/SeaGrants/node_modules/next/dist/compiled/next-server/app-page.runtime.dev.js:38:9204
    at async t7 (/Users/owocki/Documents/SeaGrants/node_modules/next/dist/compiled/next-server/app-page.runtime.dev.js:38:8898)
    at async /Users/owocki/Documents/SeaGrants/node_modules/next/dist/compiled/next-server/app-page.runtime.dev.js:38:7840
    at async Promise.all (index 0)
    at async t7 (/Users/owocki/Documents/SeaGrants/node_modules/next/dist/compiled/next-server/app-page.runtime.dev.js:38:7607)
    at async /Users/owocki/Documents/SeaGrants/node_modules/next/dist/compiled/next-server/app-page.runtime.dev.js:38:24472
TypeError: Only absolute URLs are supported
    at getNodeRequestOptions (webpack-internal:///(rsc)/./node_modules/node-fetch/lib/index.mjs:1196:15)
    at eval (webpack-internal:///(rsc)/./node_modules/node-fetch/lib/index.mjs:1292:25)
    at new Promise (<anonymous>)
    at fetch (webpack-internal:///(rsc)/./node_modules/node-fetch/lib/index.mjs:1289:12)
    at fetch (webpack-internal:///(rsc)/./node_modules/cross-fetch/dist/node-ponyfill.js:10:22)
    at eval (webpack-internal:///(rsc)/./node_modules/graphql-request/build/esm/index.js:123:22)
    at makeRequest (webpack-internal:///(rsc)/./node_modules/graphql-request/build/esm/index.js:267:28)
    at GraphQLClient.request (webpack-internal:///(rsc)/./node_modules/graphql-request/build/esm/index.js:177:16)
    at request (webpack-internal:///(rsc)/./node_modules/graphql-request/build/esm/index.js:314:19)
    at getPools (webpack-internal:///(rsc)/./src/app/page.tsx:48:95)
    at Home (webpack-internal:///(rsc)/./src/app/page.tsx:75:31)
TypeError: Only absolute URLs are supported
    at getNodeRequestOptions (webpack-internal:///(rsc)/./node_modules/node-fetch/lib/index.mjs:1196:15)
    at eval (webpack-internal:///(rsc)/./node_modules/node-fetch/lib/index.mjs:1292:25)
    at new Promise (<anonymous>)
    at fetch (webpack-internal:///(rsc)/./node_modules/node-fetch/lib/index.mjs:1289:12)
    at fetch (webpack-internal:///(rsc)/./node_modules/cross-fetch/dist/node-ponyfill.js:10:22)
    at eval (webpack-internal:///(rsc)/./node_modules/graphql-request/build/esm/index.js:123:22)
    at makeRequest (webpack-internal:///(rsc)/./node_modules/graphql-request/build/esm/index.js:267:28)
    at GraphQLClient.request (webpack-internal:///(rsc)/./node_modules/graphql-request/build/esm/index.js:177:16)
    at request (webpack-internal:///(rsc)/./node_modules/graphql-request/build/esm/index.js:314:19)
    at getPools (webpack-internal:///(rsc)/./src/app/page.tsx:48:95)
    at Home (webpack-internal:///(rsc)/./src/app/page.tsx:76:30)
Invalid projectId. Please create a unique WalletConnect Cloud projectId for your dApp https://www.rainbowkit.com/docs/installation#configure
Invalid projectId. Please create a unique WalletConnect Cloud projectId for your dApp https://www.rainbowkit.com/docs/installation#configure
Invalid projectId. Please create a unique WalletConnect Cloud projectId for your dApp https://www.rainbowkit.com/docs/installation#configure

```
even when i've got this var filled out. so not sure if thats it. maybe its not a walletconnect var?

Reply:
frontend looks like this

<img width="923" alt="Screenshot 2024-01-06 at 10 03 08 AM" src="https://github.com/allo-protocol/SeaGrants/assets/513929/ff31a6b4-9442-4cc4-8e54-351d9615adcb">
<img width="950" alt="Screenshot 2024-01-06 at 10 03 22 AM" src="https://github.com/allo-protocol/SeaGrants/assets/513929/1b0c2d7d-628a-4937-9e85-2cb2b2d658bf">
<img width="958" alt="Screenshot 2024-01-06 at 10 03 15 AM" src="https://github.com/allo-protocol/SeaGrants/assets/513929/1a86b0a7-92f3-4c28-8ac8-7ca305095648">